website/docs/r/private_endpoint.html.markdown

```diff
@@ -238,7 +238,7 @@ A `private_service_connection` block supports the following:
 
 -> **NOTE:** Some resource types (such as Storage Account) only support 1 subresource per private endpoint.
 
-* `request_message` - (Optional) A message passed to the owner of the remote resource when the private endpoint attempts to establish the connection to the remote resource. The request message can be a maximum of `140` characters in length. Only valid if `is_manual_connection` is set to `true`.
+* `request_message` - (Optional) A message passed to the owner of the remote resource when the private endpoint attempts to establish the connection to the remote resource. The request message can be a maximum of `128` characters in length. Only valid if `is_manual_connection` is set to `true`.
```

Comment:
In the validation for this field we allow up to `140` characters. Local testing confirms that this limit is valid and suggests that this limit varies depending on the service the private endpoint is connected to. Unfortunately I can't find any documentation on this

Could we instead rephrase this to 
```suggestion
* `request_message` - (Optional) A message passed to the owner of the remote resource when the private endpoint attempts to establish the connection to the remote resource. The provider allows a maximum request message length of `140` characters, however the request message maximum length is dependent on the service the private endpoint is connected to. Only valid if `is_manual_connection` is set to `true`.
```

Alternatively we can leave the description as it currently is and add a note below:

```suggestion
* `request_message` - (Optional) A message passed to the owner of the remote resource when the private endpoint attempts to establish the connection to the remote resource. The request message can be a maximum of `140` characters in length. Only valid if `is_manual_connection` is set to `true`.

-> **NOTE:** When connected to a SQL resource the `resource_message` maximum length is `128`.
```

Reply:
Personally prefer a mix between if I may have an input:

```
* `request_message` - (Optional) A message passed to the owner of the remote resource when the private endpoint attempts to establish the connection to the remote resource. The provider allows a maximum request message length of `140` characters, however the request message maximum length is dependent on the service the private endpoint is connected to. Only valid if `is_manual_connection` is set to `true`.

**EXAMPLE:** When connected to a SQL resource the `resource_message` maximum length is `128`, however MS does not provide docs for service mapping to char limit.
```

Would that work for you?

Reply:
If we can tweak the `**EXAMPLE:**` line to the following for consistency, as well as removing the last sentence since it isn't providing actionable information for the user, then that would be fine

```-> **NOTE:** When connected to a SQL resource the `resource_message` maximum length is `128`.```